README.org

```diff
@@ -70,18 +84,46 @@ is not critical for you (e.g. for a session store). Note that the database will
 not be corrupted in this case, you can just lose some write operations before
 the crash.
 
+Usage:
+
+#+BEGIN_SRC clojure
+  (ns test-db
+    (:require [konserve.memory :refer [connect-fs-store]]
+              [konserve.core :as k]))
+
+  (def my-folder "path/to/folder")
+  (def my-db (connect-fs-store my-folder))
+
+#+END_SRC
+
 *** IndexedDB
     :PROPERTIES:
     :CUSTOM_ID: h:ccbb272e-24b1-4f1e-b525-dd07c4e0e9b4
     :END:
 
+[[https://developer.mozilla.org/en-US/docs/IndexedDB][IndexedDB]] is provided as reference implementation for
+ClojureScript browser backends.
+
 For IndexedDB there is no internal JSON-representation of the underlying
```

Comment:
This is not true anymore, we actually use the `fress` library from @pkpkpk and he updated the backend.

Reply:
So this paragraph can be removed.

Reply:
Done.